node/resolvers/subscriptions/index.ts

```diff
@@ -1,11 +1,36 @@
+const SUBSCRIPTION_ORDERS_SCHEMA = "subscription_orders-v1"
```

Comment:
Some functions here can be extracted to another file named "masterDataQueryBuilder.ts" (?)

`generateTimeConstraint` => `betweenClausule(field, initDate, endDate)`
generateStatusContraint should be a function that uses `whereClausule` `andClausule` and `orClausule`, etc

check https://knexjs.org/#Builder (forsql)

this can be done in another PR,

for now, extract functions that build queries

Reply:
👍